webrender/src/resource_cache.rs

```diff
@@ -445,6 +471,23 @@ impl ResourceCache {
         }
     }
 
+    pub fn request_geometry(&mut self,
```

Comment:
I'm not sure I understand your point -- would you like to elaborate a bit?

There is a possible situation that the dimensions changed while the geometry data didn't, so if we pass the dimensions through geometry we may also need to pass the geometry data unnecessarily.